docs/static/styles/user-customizations.css

```diff
@@ -236,3 +236,59 @@
   width: 300px !important;
 }
 
+/* Links to console and trial */
+.console_links {
+  display: none;
+}
+
+
+
+@media (min-width: 768px) {
+  .console_links  {
+    float: right;
+    margin-right: 30px;
+    height: 80px;
+    display: block;
+    font-size: 0.9rem;
+    font-weight: 200;
+  }
+}
+.console_links__wrapper {
+  display:flex;
+  flex-direction:row;
+  height:100%;
+  align-items:center;
+  cursor:pointer
+}
+
+.console_links__login {
+  padding: 0 30px;
+  margin: 5px 0;
+  text-decoration: none;
+  color: #fff;
+}
+
+.console_links__login:hover,:focus {
+  text-decoration: none !important;
+  color: #fff;
+}
+
+.console_links__free-trial {
+  padding:0 15px;
+  background:#fff;
+  border:none;
+  border-radius:4px;
+  margin:5px 0;
+  color:#1f82e5;
+  text-decoration:none;
+  height:34px;
+  line-height:34px;
+  font-weight: 700;
+  transition:background-color 0.25s,color 0.25s
+}
+
+.console_links__free-trial:hover,:focus{
+  text-decoration: none !important;
+  background-color:#28d8b2;
+  color:#fff
+}
```

Comment:
I assume you copied this from the widget CSS and therefore, it should be fine. However, I've tried to compare the   prod version and the new version locally and the button shows up broken in the mobile view ( I assume it's because of the search field missing).
In the old version (with the search and in the mobile version) the button doesn't show up at all. I'm not sure if that is as intended.
<img width="381" alt="Screenshot 2021-11-15 at 10 21 47" src="https://user-images.githubusercontent.com/90635515/141757418-9bfc128a-5fda-41c9-9420-5f16b36c045c.png">
<img width="380" alt="Screenshot 2021-11-15 at 10 33 45" src="https://user-images.githubusercontent.com/90635515/141757551-f73b28ac-09b9-4f05-920e-191bff4c4381.png">

Reply:
The CSS selector was overly broad (all focus, rather than just that element). 

Also, there's no need to test it locally without search. You can use the deployed version (see the checks for the PR). That's the benefit of dogfooding :dog2: